packages/backend/src/test/fakes.ts

```diff
@@ -380,3 +382,23 @@ export const fakeCollateralAsset: CollateralAsset = {
   ),
   price: 1_000_000n,
 }
+
+export const fakePreprocessedL2TransactionsStatistics = (
+  count?: number
+): PreprocessedL2TransactionsStatistics => {
+  return {
+    depositCount: count ?? randomInt(0, 100),
```

Comment:
I think it's not correct to use random values in unit tests, as they make the tests flaky and unreliable (except for things that don't matter, like fake starkKey, etc.). They're fine in fuzz testing (but there you repeat them thousands of times). Here, come up with some random values and hardcode them.

One of the reasons you should not use randoms is that in the actual test you can't compare results to the correct expected value, but you need to *implement the logic again to have something to compare to* (e.g. for unit test for this data you actually sum these random values in the test and compare to function output). So now you can have a bug in the implementation of the logic in your unit test (so maybe write a unit test for a unit test ;)). That's why the idea is that unit tests should have mostly hardcoded values and no logic.